src/Resources/config/doctrine_mongodb_admin.xml

```diff
@@ -8,18 +8,20 @@
         <parameter key="sonata.media.admin.media.class">Sonata\MediaBundle\Admin\ODM\MediaAdmin</parameter>
         <parameter key="sonata.media.admin.media.controller">SonataMediaBundle:MediaAdmin</parameter>
         <parameter key="sonata.media.admin.media.translation_domain">SonataMediaBundle</parameter>
+        <parameter key="sonata.media.admin.media.show_mosaic_button">true</parameter>
```

Comment:
Please don't add new parameters. We should get rid of this

Reply:
How do you want we manage this option?

Reply:
Don't use a variable, use hardcoded `true` or `false` values

Reply:
Humm I'm not sure to understand. If I put an hardcoded value in the show_mosaic_button option, how the user will choose if he/she wants to show or hide the button?